### ISSUE TITLE
[BUG] gi.repository.Gtk has no attribute Template

### ISSUE THREAD

**Issue by hthre7** [bottlesdevs/Bottles#128]
**Describe the bug**
app will not open show error when open in terminal
error down

**Installation**
 - Method: Appimage
 - Version [2.1.2]

**Desktop (please complete the following information):**
 - OS: Linux Mint
 - Version [19.3]

**Additional context**
error : 
```
  File "/tmp/.mount_BottleDf0tuj/usr/bin/bottles", line 49, in <module>
    from bottles import main
  File "/tmp/.mount_BottleDf0tuj//usr/local/share/bottles/bottles/main.py", line 48, in <module>
    from .window import BottlesWindow
  File "/tmp/.mount_BottleDf0tuj//usr/local/share/bottles/bottles/window.py", line 26, in <module>
    from .runner import BottlesRunner
  File "/tmp/.mount_BottleDf0tuj//usr/local/share/bottles/bottles/runner.py", line 35, in <module>
    from .download import DownloadManager
  File "/tmp/.mount_BottleDf0tuj//usr/local/share/bottles/bottles/download.py", line 24, in <module>
    @Gtk.Template(resource_path='/com/usebottles/bottles/download-entry.ui')
  File "/usr/lib/python3/dist-packages/gi/overrides/__init__.py", line 39, in __getattr__
    return getattr(self._introspection_module, name)
  File "/usr/lib/python3/dist-packages/gi/module.py", line 137, in __getattr__
    self.__name__, name))
AttributeError: 'gi.repository.Gtk' object has no attribute 'Template'
```

**Comment by mirkobrombin** [comment]
I think (from what I understand from your post) that the error is given by a too old version of Gtk. Maybe Mint offers a slightly updated version?

**Comment by hthre7** [comment]
**apt show python3-gi**
``` 
apt show python3-gi
Package: python3-gi
Version: 3.26.1-2ubuntu1
Priority: important
Section: python
Source: pygobject
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Debian GNOME Maintainers <pkg-gnome-maintainers@lists.alioth.debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 645 kB
Depends: python3 (<< 3.7), python3 (>= 3.6~), python3:any (>= 3.3.2-2~), libc6 (>= 2.14), libffi6 (>= 3.0.4), libgirepository-1.0-1 (>= 1.44.0), libglib2.0-0 (>= 2.41.1), gir1.2-glib-2.0 (>= 1.39.0)
Homepage: https://wiki.gnome.org/Projects/PyGObject
Task: minimal
Supported: 5y
Download-Size: 153 kB
APT-Manual-Installed: yes
APT-Sources: http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages
Description: Python 3 bindings for gobject-introspection libraries
 GObject is an abstraction layer that allows programming with an object
 paradigm that is compatible with many languages. It is a part of Glib,
 the core library used to build GTK+ and GNOME.
```
**apt show python3**
```
apt show python3
Package: python3
Version: 3.6.7-1~18.04
Priority: important
Section: python
Source: python3-defaults
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Matthias Klose <doko@debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 191 kB
Provides: python3-profiler
Pre-Depends: python3-minimal (= 3.6.7-1~18.04)
Depends: python3.6 (>= 3.6.7-1~), libpython3-stdlib (= 3.6.7-1~18.04)
Suggests: python3-doc (>= 3.6.7-1~18.04), python3-tk (>= 3.6.7-1~), python3-venv (>= 3.6.7-1~18.04)
Replaces: python3-minimal (<< 3.1.2-2)
Homepage: https://www.python.org/
Task: minimal, ubuntu-core
Cnf-Extra-Commands: python
Cnf-Priority-Bonus: 5
Supported: 5y
Download-Size: 47.2 kB
APT-Manual-Installed: yes
APT-Sources: http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages
Description: interactive high-level object-oriented language (default python3 version)
 Python, the high-level, interactive object oriented language,
 includes an extensive class library with lots of goodies for
 network programming, system administration, sounds and graphics.
 .
 This package is a dependency package, which depends on Debian's default
 Python 3 version (currently v3.6).
```

**Comment by hthre7** [comment]
is it possible to pack newer version of  python in the appimage

**Comment by mirkobrombin** [comment]
Maybe also python-gi should be packed

**Comment by hthre7** [comment]
is possible to add in extracted appimage the repacked with knew python and python-gi

**Comment by mirkobrombin** [comment]
> is possible to add those packages in extracted AppImage then repack with new python and python-gi

No. I have to tweak the build process to integrate the necessary dependencies

**Comment by mirkobrombin** [comment]
Should be fixed with 3.0.6 that bundle the GTK dependencies.
